concepticondata/conceptlists/Hartmann-2013-162.tsv

```diff
@@ -0,0 +1,163 @@
+Hartmann-2013-162-100	100	LONG FOR	3473	LONG (FOR SOMETHING)	The teacher longs for coffee.	A longs for P	New meaning	long-for
+Hartmann-2013-162-101	101	LOOK AT	1819	LOOK	The boy looked at the girl.	A looks at P	Core list	look-at
+Hartmann-2013-162-102	102	MAKE	734	MAKE	The man made a box out of wood.	A makes P (out of X)	Old meaning	make
+Hartmann-2013-162-103	103	MAKE APPEAR			The wizard made a white dove appear.	A makes P appear	New meaning	make-appear
+Hartmann-2013-162-104	104	MAKE CLIMB			The man made the monkey climb the wall.	C makes A climb L	New meaning	make-climb
+Hartmann-2013-162-105	105	MAKE HUNGRY			The hard work made the men hungry.	M makes E hungry	New meaning	make-hungry
+Hartmann-2013-162-106	106	MAKE ONE'S PRESENCE KNOWN			He made his presence known in the province.	S makes one's presence known	New meaning	make-ones-presence-known
+Hartmann-2013-162-107	107	MAKE RUN			The girl made the horse run.	A makes P run	New meaning	make-run
+Hartmann-2013-162-108	108	MAKE SICK			The cold made the girl sick.	M makes E sick	New meaning	make-sick
+Hartmann-2013-162-109	109	MARRY	203	MARRY	The man married the woman.	X marries Y	New meaning	marry
+Hartmann-2013-162-110	110	MEET	1831	MEET	The men met the boys.	A meets X	Core list	meet
+Hartmann-2013-162-111	111	NAME	1405	NAME	The parents called the baby Anna.	A name X (a) Y	Core list	name
+Hartmann-2013-162-112	112	OFFER			The girl offerd her help to a stranger.	X offers Y to Z	New meaning	offer
+Hartmann-2013-162-113	113	PEEL	275	PEEL	The boy peeled the bark off the stick.	A peels (X off) P	Core list	peel
+Hartmann-2013-162-114	114	PLAY	1413	PLAY	The child is playing.	S plays	Core list	play
+Hartmann-2013-162-115	115	POUR	1487	POUR	The man poured water into the glass.	A pours T somewhere (L)	Core list	pour
+Hartmann-2013-162-116	116	PROPOSE (a marriage)			The man proposed to the woman.	A proposes to M	New meaning	propose-a-marriage
+Hartmann-2013-162-117	117	PUSH	1452	PUSH	The boy pushed the girl (into the water).	A pushes P (somewhere (L))	Extended list	push
+Hartmann-2013-162-118	118	PUT	998	PUT	I put the cup onto the table.	A puts T somewhere (L)	Core list	put
+Hartmann-2013-162-119	119	RAIN	1253	RAIN (RAINING)	It rained.	(it) rains	Core list	rain
+Hartmann-2013-162-120	120	READ	724	READ	The girl is reading a book.	X reads Y	New meaning	read
+Hartmann-2013-162-121	121	REFUSE	49	REFUSE	The old lady refused the man a place to sleep.	X refuses Y Z	New meaning	refuse
+Hartmann-2013-162-122	122	REMAIN	987	REMAIN	The woman remained in the village.	A remains in L	New meaning	remain
+Hartmann-2013-162-123	123	REMEMBER	707	REMEMBER	She remembered the street.	A remembers X	Old meaning	remember
+Hartmann-2013-162-124	124	REST	168	REST	The girl was resting.	X rests	New meaning	rest
+Hartmann-2013-162-125	125	ROLL	827	ROLL	The ball is rolling.	A rolls	Core list	roll
+Hartmann-2013-162-126	126	ROLL (tr)			The child rolls the ball.	A rolls P	New meaning	roll-tr
```

Comment:
I think the concept 827 ROLL is transitive.

Reply:
The definition we have is rather ambiguous and perhaps will need to be refined. But the Wordnet definition serving as supplementary metadata to this concept suggests that it's supposed to be transitive.